MANIFEST.in

```diff
@@ -1,9 +1,10 @@
-include apache-2.0.LICENSE
+include LICENSE
```

Comment:
apache-2.0.LICENSE is what we use in all other projects... any reason for changing the name?

Reply:
I'm annoyed by the lack of support for this convention in GitHub. I will revert to the nexB standard.

Reply:
Ah... good point... What about adding a symlink from LICENSE to apache-2.0.LICENSE?